### ISSUE TITLE
Run lifecycle storage smoke against private HF Bucket

### ISSUE THREAD

**Issue by bie3yeik-lgtm** [bie3yeik-lgtm/jpapt-v2.2-inspection#71]
## Goal

Empirically prove the lifecycle persistence layout against a real private Hugging Face Bucket without running model inference, GPU compute, Docker evaluation, or HF Jobs.

## Existing implementation

The repository already contains the manual-only workflow:

```text
.github/workflows/candidate-lifecycle-storage-smoke.yml
Candidate Lifecycle Storage Smoke
```

It requires `confirm_write=true`, prepares the selected lifecycle Bucket with `--exist-ok --private`, generates a unique synthetic request/execution identity, persists it through the canonical `scripts/ci/persist-candidate-lifecycle.sh`, and reads back:

```text
request-level event
request-level running state
execution-level event
execution-level running state
```

The workflow validates each readback against `CandidateRequestLifecycleV1` and checks canonical observation SHA-256 equality with the source snapshot.

## Why this remains open

Static/fake-client contract CI proves path construction and local semantics but is not evidence that the configured `HF_TOKEN` can create/write/read the actual Bucket. The current GitHub connector cannot start an arbitrary `workflow_dispatch` run or inspect secret values, so this operational proof must be triggered from GitHub Actions by an authorized operator.

## Acceptance criteria

Run `Candidate Lifecycle Storage Smoke` with:

```text
confirm_write = true
lifecycle_bucket = <explicit private test lifecycle Bucket, or the configured/inferred lifecycle Bucket>
```

and verify:

- workflow concludes successfully;
- Bucket creation/reuse succeeds as private;
- request-level event readback succeeds;
- request-level state readback succeeds;
- execution-level event readback succeeds;
- execution-level state readback succeeds;
- all four readbacks validate as lifecycle snapshots;
- all four canonical observation hashes equal the generated source snapshot;
- the Actions artifact `candidate-lifecycle-storage-smoke-<run>-<attempt>` is preserved;
- no model inference, GPU job, Docker evaluation, or HF Jobs workload was launched.

## Evidence to record

After success, add the workflow run ID/URL and the synthetic `request_id`, `request_execution_id`, `request_key`, `execution_key`, and lifecycle Bucket name to this issue. Then this issue can be closed as completed.

The synthetic Bucket objects are intentionally uniquely keyed and may remain as auditable write/read evidence unless a separate retention policy is adopted.

**Comment by bie3yeik-lgtm** [comment]
Progress update (2026-08-18): lifecycle persistence contracts are now green after aligning them with the current materialized-state implementation and Rust protocol authority.

Verified local/CI evidence:
- request-level and execution-level persistence layout contracts are green;
- materialized state uses `materialize_state` with retry-aware monotonic comparison;
- canonical rejection/receipt/ACK evidence is validated with Rust before persistence;
- acknowledged persistence requires both receipt and ACK and revalidates their binding;
- authority contract now triggers when `scripts/ci/persist-candidate-lifecycle.sh` itself changes;
- full revalidation on PR #84 succeeded: Persist Contracts 32131531384, Protocol Authority 32131531379, Execution Storage 32131531450, Validate HF Layout 32131531386.

This still does NOT satisfy this issue. A real private Hugging Face Bucket create/write/read round trip with the configured `HF_TOKEN` is still required via the manual `Candidate Lifecycle Storage Smoke` workflow. No remote write was claimed from the contract runs above.

**Comment by bie3yeik-lgtm** [comment]
2026-08-19 execution-capability recheck: this smoke remains operationally blocked, not source-blocked. The GitHub connector available here cannot start an arbitrary `workflow_dispatch` run, and the connected Hugging Face tool surface exposes Hub metadata/Jobs but no Bucket create/write/read operation. The HF Jobs connector is also currently failing read-only `ps --all` response decoding, and I will not launch paid compute as a workaround. Therefore the existing manual-only `Candidate Lifecycle Storage Smoke` remains the correct zero-model-compute proof path once an authorized operator can trigger it with `confirm_write=true` against an explicit private test Bucket.

**Comment by bie3yeik-lgtm** [comment]
2026-08-19 lifecycle storage smoke capability recheck:

- Hugging Face authentication is currently valid as personal account `gawohok7` with the available OAuth scopes including `jobs`, `read-mcp`, and `read-repos`.
- The connected Hugging Face toolset still exposes Jobs/model/dataset/space/doc operations, but no direct Bucket create/write/read action that can execute this storage smoke.
- The GitHub connector can inspect workflow runs and repository content but still does not expose an arbitrary `workflow_dispatch` start action for the existing manual `Candidate Lifecycle Storage Smoke` workflow.

Therefore there is still no safe execution path in this session for the required real private-Bucket create/write/read proof. Do not substitute a paid HF Job, do not change the workflow to leak secrets into a PR/push path, and do not claim the fake-client/static contracts as real Bucket evidence.

Keep #71 open until an authorized operator can trigger the existing manual workflow with `confirm_write=true`, or the toolset gains an equivalent direct Bucket lifecycle capability.

**Comment by bie3yeik-lgtm** [comment]
Operational acceptance evidence completed on 2026-08-19.

Workflow:
- `Agent Lifecycle Storage Smoke`
- run: `32230245637`
- attempt: `1`
- conclusion: `success`
- exact branch head: `bdeec420d6a5fbe1eeac87cf19a3ac7730761b72`
- validation PR: #167 (validation-only; not for merge)

Private lifecycle Bucket:
- `gawohok7/jpapt-v2.2-inspection-lifecycle`

Synthetic identity:
- `request_id = storage-smoke-32230245637-1`
- `request_execution_id = storage-smoke-exec-32230245637-1`
- `request_key = 42141f17f1453d75af49b727`
- `execution_key = 0b3eb3874aa2fa8fa45c18b4`
- event key: `evaluation-32230245637-1-running-667fcf2c94c3befe`
- canonical observation SHA-256: `667fcf2c94c3befe3dce6033526fd559331c7b9a271c5d750ddb5842fc79195a`

Verified remote write/read paths:
1. request-level event
2. request-level `running` state
3. execution-level event
4. execution-level `running` state

All four remote readbacks were validated by `build-candidate-request-lifecycle.py --validate`, and all canonical observation hashes matched the generated source snapshot.

Artifact:
- name: `candidate-lifecycle-storage-smoke-32230245637-1`
- artifact ID: `9356969724`
- archive SHA-256: `693251a773341ff7e9fcfe20c03a8a3940e7b640eb15d2c2be42b74174ec3e24`
- retention: 14 days

No model inference, GPU workload, Docker evaluation, or Hugging Face Job was launched. This satisfies the acceptance criteria for the real private HF Bucket lifecycle storage smoke.
